mne/channels/montage.py

```diff
@@ -669,9 +730,9 @@ def _get_scaling(unit, scale):
         return scale[unit]
 
 
-def read_dig_montage(hsp=None, hpi=None, elp=None, point_names=None,
-                     unit='auto', fif=None, egi=None, bvct=None,
-                     transform=True, dev_head_t=False, ):
+def read_dig_montage(hsp=NEVER_USED_PARAM, hpi=None, elp=None,
```

Comment:
why hsp=NEVER_USED_PARAM ???

Reply:
yes I messed the hsp. I was tracking hpi. I did this just in case that somewhere there was hpi=hpi and I didn't want to overwrite it.